posthog/temporal/data_imports/sources/stripe/stripe.py

```diff
@@ -78,13 +95,28 @@ def get_rows():
         incremental_field_config = INCREMENTAL_FIELDS.get(endpoint, [])
         incremental_field_name = incremental_field_config[0]["field"] if incremental_field_config else "created"
 
-        if not should_use_incremental_field or (
-            db_incremental_field_last_value is None and db_incremental_field_earliest_value is None
+        if (
+            not should_use_incremental_field
+            or (db_incremental_field_last_value is None and db_incremental_field_earliest_value is None)
+            or isinstance(resource, StripeNestedResource)
```

Comment:
So nested resources can't ever be set as append-only? If so, then you'll need to change this too: https://github.com/PostHog/posthog/blob/master/posthog/temporal/data_imports/sources/stripe/source.py#L71